### ISSUE TITLE
GFSSF acceptance validator fails with IndexError

### ISSUE THREAD

**Issue by micahjsmith** [HDI-Project/ballet#33]
* ballet version: 0.5.3-dev (b413e30)
* Python version: 3.6.3
* Operating System: Ubuntu 14.04.5

### Description

GFSSF validator fails judging example feature

### What I Did

```
[2019-03-20 13:03:26,350] {ballet: gfssf_validator.py:60} INFO - Judging Feature using GFSSF: lambda_1=0.011564066054355004, lambda_2=0.011564066054355004
[2019-03-20 13:03:26,350] {ballet: gfssf_validator.py:65} DEBUG - Testing with omitted feature: None
[2019-03-20 13:03:26,374] {ballet: log.py:115} INFO - Ballet Validation: evaluating feature performance...FAILURE
Traceback (most recent call last):
  File "./validate.py", line 11, in <module>
    ballet.validation.main(ames)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/ballet/validation/__init__.py", line 154, in main
    evaluate_feature_performance(project)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/funcy/decorators.py", line 38, in wrapper
    return deco(call, *dargs, **dkwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/ballet/validation/__init__.py", line 25, in validation_stage
    return call()
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/funcy/flow.py", line 38, in wrapper
    return func(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/ballet/util/log.py", line 145, in wrapper
    return func(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/funcy/decorators.py", line 55, in __call__
    return self._func(*self._args, **self._kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/ballet/validation/__init__.py", line 122, in evaluate_feature_performance
    accepted = evaluator.judge(proposed_feature)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/ballet/validation/gfssf_validator.py", line 67, in judge
    cmi = estimate_conditional_information(feature_df, self.y, z)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/ballet/validation/entropy.py", line 270, in estimate_conditional_information
    h_xyz = estimate_entropy(xyz, epsilon)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/ballet/validation/entropy.py", line 186, in estimate_entropy
    selected_cont_samples = cont_features[unique_mask.ravel(), :]
IndexError: boolean index did not match indexed array along dimension 0; dimension is 2930 but corresponding boolean dimension is 5860
The command "./validate.py" exited with 1.
```

See https://travis-ci.org/micahjsmith/ballet-ames-demo/jobs/508915068

**Comment by kelvin-lu** [comment]
Thanks for leaving this! So far, I've tried replicating this bug in a Jupyter notebook, by reimplementing your feature and then judging it on the GFSSF validator (which it passes). It seems to work fine in this case, so I'm currently looking to see if it might be related to project collection.

**Comment by micahjsmith** [comment]
Closed by #35
